controllers/openstackcluster_controller_test.go

```diff
@@ -233,14 +233,14 @@ var _ = Describe("OpenStackCluster controller", func() {
 		networkClientRecorder := mockScopeFactory.NetworkClient.EXPECT()
 
 		// Fetch external network
-		networkClientRecorder.ListNetwork(networks.ListOpts{
+		/*networkClientRecorder.ListNetwork(networks.ListOpts{
```

Comment:
?

Reply:
According to the Settings in the testCluster.Spec(line 218: DisableAPIServerFloatingIP is true and there is no NodeCIDR ), there is no need to check the externalNetwork, it will not execute ReconcileExternalNetwork(openStackCluster)

Reply:
Got it. Please can you remove this code instead of commenting it out?